packages/create-sheriff-config/src/utils/setSheriffConfig.ts

```diff
@@ -1,12 +1,13 @@
 import { highlight } from 'cli-highlight';
 import { consola } from 'consola';
 import { sheriffStartingOptions } from '@sherifforg/constants';
+import type { SheriffConfigurablePlugins } from '../../../sheriff-types/src';
```

Comment:
the correct import is `@sherifforg/types`

Reply:
🤦‍♂️